### PR TITLE
mgmt, local generation for Network 2022-06-01

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to [issue#4810](https://github.com/Azure/sdk-release-request/issues/4810)

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by v-hongli1** [comment]
The classes `PropagatedRouteTableNfv`, `RoutingConfigurationNfv`,` RoutingConfigurationNfvSubResource` have been removed, so add `<revapi.skip>true</revapi.skip>` in `pom.xml`.

**Comment by v-hongli1** [comment]
In [PR#25962](https://github.com/Azure/azure-rest-api-specs/pull/25962) the `RoutingConfigurationNfv` was renamed to `RoutingConfiguration`,  the `PropagatedRouteTableNfv` and `RoutingConfigurationNfvSubResource` were removed.

**Comment by weidongxu-microsoft** [comment]
seems from here https://github.com/Azure/azure-rest-api-specs/pull/25962 (please try find this kind of breaking change from service PR)

Please update the changelog about the removed classes.

File: sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ### Features Added
 
 - The class `RoutingConfigurationNfv` renamed to `RoutingConfiguration`.
-- Remove classes `PropagatedRouteTableNfv` and `RoutingConfigurationNfvSubResource`.
+- The classes `PropagatedRouteTableNfv` and `RoutingConfigurationNfvSubResource` removed.
```
Comment (haolingdong-msft) [inline sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md:8]: These are breaking changes. Please add back breaking changes section and put them under breaking changes section.

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md]: Fixed in the new version.

File: sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md [hunk 2]
```diff
@@ -2,14 +2,17 @@
 
 ## 2.34.0-beta.1 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
+- The class `RoutingConfigurationNfv` renamed to `RoutingConfiguration`.
+- The classes `PropagatedRouteTableNfv` and `RoutingConfigurationNfvSubResource` were removed.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md:8]:
```suggestion
- Renamed class `RoutingConfigurationNfv` to `RoutingConfiguration`.
- Removed classes `PropagatedRouteTableNfv` and `RoutingConfigurationNfvSubResource`.
```

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md:8]: Fixed in the new version.

**Review by weidongxu-microsoft — APPROVED** [review]

**Review by haolingdong-msft — APPROVED** [review]

**Comment by XiaofeiCao** [comment]
`RoutingConfiguration` already exists? 
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/RoutingConfiguration.java

Seems to me they are removing nfv(don't know what that is) related models. Perhaps in changelog, just add `RoutingConfiguration` to the removed line?

**Comment by v-hongli1** [comment]
@XiaofeiCao 
The `RoutingConfiguration` model only appears in the "2023-06-01" version. I think the `RoutingConfigurationNfv` model has been replaced by `RoutingConfiguration`, since `PropagatedRouteTableNfv` and `RoutingConfigurationNfvSubResource` are related models of `RoutingConfigurationNfv`, so they were also removed.

**Comment by weidongxu-microsoft** [comment]
> `RoutingConfiguration` already exists? https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/RoutingConfiguration.java
> 
> Seems to me they are removing nfv(don't know what that is) related models. Perhaps in changelog, just add `RoutingConfigurationNfv` to the removed line?

But there IS an existing `RoutingConfiguration` class before your PR?

**Comment by v-hongli1** [comment]
> `RoutingConfiguration` already exists? https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/RoutingConfiguration.java
> 
> Seems to me they are removing nfv(don't know what that is) related models. Perhaps in changelog, just add `RoutingConfiguration` to the removed line?

The `RoutingConfiguration` model only appears in the `2023-06-01` version. I think the `RoutingConfigurationNfv` model has been replaced by `RoutingConfiguration`, since `PropagatedRouteTableNfv` and `RoutingConfigurationNfvSubResource` are related models of `RoutingConfigurationNfv`, so they were also removed. So should it be described as `renamed` or `removed` in `Breaking Changes`?

**Comment by weidongxu-microsoft** [comment]
What are you talking about?

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2023-05-01/virtualWan.json#L8803-L8827

Please give correct information.

**Comment by v-hongli1** [comment]
> `RoutingConfiguration` already exists? https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/RoutingConfiguration.java
> 
> Seems to me they are removing nfv(don't know what that is) related models. Perhaps in changelog, just add `RoutingConfiguration` to the removed line?

As per https://github.com/Azure/azure-rest-api-specs/pull/25962#issuecomment-1739864700, I think it is more appropriate to describe ```- Removed class `RoutingConfigurationNfv`, and used class `RoutingConfiguration` to maintain consistency of routing configuration properties across all connections.``` in `Breaking Changes`.

**Comment by weidongxu-microsoft** [comment]
> > `RoutingConfiguration` already exists? https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/RoutingConfiguration.java
> > Seems to me they are removing nfv(don't know what that is) related models. Perhaps in changelog, just add `RoutingConfiguration` to the removed line?
> 
> As per [Azure/azure-rest-api-specs#25962 (comment)](https://github.com/Azure/azure-rest-api-specs/pull/25962#issuecomment-1739864700), I think it is more appropriate to describe `` - Removed class `RoutingConfigurationNfv`, and used class `RoutingConfiguration` to maintain consistency of routing configuration properties across all connections. `` in `Breaking Changes`.

What is the API that previous work with `RoutingConfigurationNfv`? Does it now work with `RoutingConfiguration` instead?

**Comment by XiaofeiCao** [comment]
> As per [Azure/azure-rest-api-specs#25962 (comment)](https://github.com/Azure/azure-rest-api-specs/pull/25962#issuecomment-1739864700), I think it is more appropriate to describe `` - Removed class `RoutingConfigurationNfv`, and used class `RoutingConfiguration` to maintain consistency of routing configuration properties across all connections. `` in `Breaking Changes`.

There is also [`PropagatedRouteTable`](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2023-05-01/virtualWan.json#L8828-L8846) as well. If we want to mention the reason, we probably want to put these three removed classed together.

**Comment by v-hongli1** [comment]
> > > `RoutingConfiguration` already exists? https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/RoutingConfiguration.java
> > > Seems to me they are removing nfv(don't know what that is) related models. Perhaps in changelog, just add `RoutingConfiguration` to the removed line?
> > 
> > 
> > As per [Azure/azure-rest-api-specs#25962 (comment)](https://github.com/Azure/azure-rest-api-specs/pull/25962#issuecomment-1739864700), I think it is more appropriate to describe `` - Removed class `RoutingConfigurationNfv`, and used class `RoutingConfiguration` to maintain consistency of routing configuration properties across all connections. `` in `Breaking Changes`.
> 
> What is the API that previous work with `RoutingConfigurationNfv`? Does it now work with `RoutingConfiguration` instead?

The `RoutingConfigurationNfv` is only referenced in `NetworkVirtualApplianceConnectionsClient`.

**Comment by weidongxu-microsoft** [comment]
Then what does use `NetworkVirtualApplianceConnectionsClient` use now?

**Comment by v-hongli1** [comment]
> Then what does `NetworkVirtualApplianceConnectionsClient` use now?

Now `NetworkVirtualApplianceConnectionsClient` uses `RoutingConfiguration` like other clients such as `HubVirtualNetworkConnectionsClient`.

**Review by XiaofeiCao — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
OK, then it makes sense.
```
- Removed class `RoutingConfigurationNfv`. Please used class `RoutingConfiguration`.
```

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**